### PR TITLE
[feat] Slider component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created slider component, which provides users with ability to enter their input by selecting from a pre-existing range, by sliding the head to their desired position.

Also updated the sidebar navigation to reflect the addition of slider component documentation page.

Deploy link: [Slider component](https://deploy-preview-43--powertrain-ui.netlify.app/documentation/html/slider.html)

image

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Excellent code structure, Everything looks fine. Good to go from my side.

File: Documentation/css/slider.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@import url("../../components/slider/slider.css");
+
+.example {
+  width: 100%;
+  margin: var(--size-9) 0;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 16px;
```
Comment (RakshithBhat03) [inline Documentation/css/slider.css]: You can use rem unit here or maybe a sizing variable if it's defined.

File: Documentation/css/slider.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  padding: var(--size-13);
+  border: 2px solid var(--secondary-color);
+  border-radius: 10px;
```
Comment (RakshithBhat03) [inline Documentation/css/slider.css]: You can use a variable here if it's defined.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested minor changes. Everything else looks good.

**Review by junaidshaikh-js — APPROVED** [review]
Clean code. All the semantic elements are used.

File: Documentation/css/slider.css (hunk 2 above)
Comment (Hamdrive) [reply Documentation/css/slider.css]: Yes I will update it

File: Documentation/css/slider.css (hunk 1 above)
Comment (Hamdrive) [reply Documentation/css/slider.css]: gap property might not be needed here, will refactor it

**Comment by Hamdrive** [comment]
Succesfully reviewed component, have implemented requested change.
Merging to dev and closing branch.

**Pull request merged**